### ISSUE TITLE
[a11y]: Breadcrumbs don't inform screen reader about current page

### ISSUE THREAD

**Issue by simon-burns** [carbon-design-system/carbon#18113]
### Package

@carbon/react

### Package version

1.70.0

### Automated testing tool and ruleset

AVT3 - JAWS screen reader

### Assistive technology

JAWS

### Description

The breadcrumbs component should inform screen reader users which is the current page in the breadcrumbs.

### WCAG 2.1 Violation

4.1.2 Name, Role, Value

### Reproduction/example

https://stackblitz.com/edit/github-dupbrt?file=src%2FApp.jsx

### Steps to reproduce

Use a screen reader with a Breadcrumb component, and it will not distinguish which breadcrumb item is for the current page making it more difficult for a screen reader user to understand where they are.

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems
